src/main/webapp/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<!-- 2019 Capstone Project -->
+<!-- Jenny Franklin, James Willhoite, Matthew Thomas -->
+
+<head>
+    <link rel="stylesheet"
+          type="text/css"
+          href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.css">
+
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"
+            integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+            crossorigin="anonymous">
+    </script>
+
+    <script src="semantic/dist/semantic.min.js"></script>
```

Comment:
This is a relative link, is this going to be hosted on the server?

Reply:
Sorry, I didn't see this. I need to ask about that, as I think it should be in a folder on the server. I'm currently a little foggy on exactly what I'm calling, but it works (?) When I figure that out, I'm going to download it so I can go in and edit where needed. Does that answer your question?

Reply:
Everything else looks good on this code. Just this semantic.min.js script. It would be faster to host on the server, but it can be a cdn.